config.toml

```diff
@@ -132,6 +132,10 @@ websocket_addr = 'ws://127.0.0.1:26657/websocket'
 # Hermes uses a large preconfigured timeout (on the order of minutes).
 rpc_timeout = '10s'
 
+# Enable verification of headers including in `ClientUpdate` message using the light client.
+# Default: true
+verify_headers = true
+
```

Comment:
Maybe mention this is experimental. Also maybe:
```suggestion
# Whether or not the full node is trusted.
# If not trusted it enables verification of headers including in `ClientUpdate` message using the light client.
# Note: If the full node is configured as trusted then in addition to headers not being verified, the verification traces will not be provided. This may cause failure in client updates after significant change in validator sets.
# Default: false
trusted_node = false

```

Reply:
[ac17e4b](https://github.com/informalsystems/hermes/pull/3328/commits/ac17e4b2af131df84ce254f91636da4735a4f9fd)